packages/interledger/src/admin-api/resolvers/ilpAccount.ts

```diff
@@ -29,31 +32,87 @@ export const getIlpAccount: QueryResolvers['ilpAccount'] = async (
   args,
   ctx
 ): ResolversTypes['IlpAccount'] => {
-  // TODO: get account all information from accounts service
-  const account = await ctx.accountsService.getAccount(args.id)
-  return {
-    id: account.accountId
-  }
+  return await ctx.accountsService.getAccount(args.id)
 }
 
 export const createIlpAccount: MutationResolvers['createIlpAccount'] = async (
   parent,
   args,
   ctx
 ): ResolversTypes['CreateIlpAccountMutationResponse'] => {
-  // TODO:
-  console.log(ctx) // temporary to pass linting
-  return {}
+  try {
+    const accountOrError = await ctx.accountsService.createAccount(args.input)
+    if (isCreateAccountError(accountOrError)) {
+      switch (accountOrError) {
+        case CreateAccountError.DuplicateAccountId:
+          return {
+            code: '409',
```

Comment:
Why return a numeric error code (`409`) instead of the more descriptive `DuplicateAccountId`?

Reply:
I'd only ever seen us use http status codes and saw https://github.com/interledger/rafiki/pull/61#discussion_r678579945